src/shared/services/money.js

```diff
@@ -60,7 +60,11 @@ function refresh() {
   }
   const url = isClientSide() ? '/community-app-assets/api/exchange-rates'
     : `${OE_API}/latest.json?app_id=${OE_TOKEN}`;
-  return fetch(url).then(res => res.json()).then((res) => {
+  return fetch(url, {
+    headers: {
+      authorization: process.env.SERVER_API_KEY
```

Comment:
```
Authorization: `ApiKey ${process.env.SERVER_API_KEY}`
```
To follow the standard look of the Authorization header, where the key or token is always prepended by the auth scheme name. Also requires corresponding update of the auth check method.